ui/app/components/app/alerts/unconnected-account-alert/unconnected-account-alert.js

```diff
@@ -56,7 +67,27 @@ const SwitchToUnconnectedAccountAlert = () => {
         </>
       )}
       footerClassName="unconnected-account-alert__footer"
-    />
+    >
+      <Checkbox
+        id="unconnectedAccount_dontShowThisAgain"
```

Comment:
This prop doesn't yet exist in `CheckBox`. I should make it exist.

Reply:
This was done in #8553

Reply:
I'm not going to consider this PR to be strictly dependant upon #8553 because this still works fine as-is, kinda. The prop is useless, and this is sub-optimal from an accessibility standpoint, but it's no worse than the rest of our extension. Plus #8553 will likely get merged soon anyway.